alpha/apps/kaltura/lib/myPlaylistUtils.class.php

```diff
@@ -1132,14 +1132,19 @@ protected static function getCaptionFilePath($captionAsset, &$localFilePath, &$s
 
 	/**
 	 * @param $entryIds
+	 * @param $readyOnly
 	 * @return array
 	 * @throws PropelException
 	 */
-	public static function getEntryIdsCaptions($entryIds)
+	public static function getEntryIdsCaptions($entryIds, $readyOnly = false)
```

Comment:
to make this more generic lets pass array of statuses
and it has values that we add status in

Reply:
done